engine/src/main/java/com/redhat/cloud/notifications/processors/SystemEndpointTypeProcessor.java

```diff
@@ -40,4 +40,24 @@ protected Set<User> getRecipientList(Event event, List<Endpoint> endpoints, Emai
 
         return recipientResolver.recipientUsers(event.getOrgId(), requests, subscribers, emailSubscriptionType.isOptIn());
     }
+
+    /**
+     * Gets a set of the email subscribers which should get an email for the
+     * given event.
+     * @param event the event the users are subscribed to.
+     * @return a set of {@link java.util.UUID}s in the {@link String} shape.
+     */
+    protected Set<String> getSubscribersUUIDList(final Event event) {
+        final EventType eventType = event.getEventType();
+
+        final List<String> subscribers = this.emailSubscriptionRepository.getEmailSubscribersUserId(
+            event.getOrgId(),
+            eventType.getApplication().getBundle().getName(),
+            eventType.getApplication().getName(),
+            eventType.getName(),
+            EmailSubscriptionType.INSTANT
+        );
+
+        return Set.copyOf(subscribers);
```

Comment:
Is the copy needed?

Reply:
No... Maybe I should return the list directly, right?

Reply:
Changed to a list!